### ISSUE TITLE
Pipeline run fails to start for KF 1.1.0 multi-user on-prem installation

### ISSUE THREAD

**Issue by ConverJens** [kubeflow/kubeflow#5273]
/kind bug

**What steps did you take and what happened:**
Deployed KF successfully. Initiating a run and pods get stuck in a pending state. Logs show:
`Warning  FailedMount  6s (x50 over 85m)    kubelet   MountVolume.SetUp failed for volume "mlpipeline-minio-artifact" : secret "mlpipeline-minio-artifact" not found`

**What did you expect to happen:**
Pipeline to run

**Anything else you would like to add:**
The secret is present in the ns "kubeflow" but since this is multi-user the pipeline launches in the user ns, "admin".

**Environment:**
- Kubeflow version: 1.1.0
- kfctl version: kfctl v1.1.0-0-g9a3621e
- Kubernetes platform: rancher k8s
- Kubernetes version: v1.17.6:
- OS ubuntu 18.04

**Comment by mtszkw** [comment]
Unfortunately this issue was closed with no dicussion, but in case anybody is here looking for the solution: I had the same problem ([v. 1.0.4](https://www.github.com/kubeflow/pipelines/commit/b604c6171244cc1cd80bfdc46248eaebf5f985d6)), which I finally managed to solve. With the secret present in "kubeflow" ns I had to copy it to my user ns (the one you use in KF pipeline) using this command https://stackoverflow.com/questions/55515594/is-there-a-way-to-share-a-configmap-in-kubernetes-between-namespaces/55516643

`kubectl get secret mlpipeline-minio-artifac --namespace=kubeflow  -o yaml | sed 's/namespace: kubeflow/namespace: <to-namespace>/' | kubectl create -f -`

After this, the next pipeline run is all green.
